tests/Directions/ClientTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace GoogleMapsClient\Tests\Directions;
+
+use GoogleMapsClient\GoogleMapsRequest;
+use GoogleMapsClient\Tests\AbstractClientTest;
+use GuzzleHttp\Psr7\Response;
+
+class ClientTest extends AbstractClientTest
+{
+    public function testOk(): void
+    {
+        $apiResult = require __DIR__ . '/DirectionApiResult.php';
```

Comment:
Another option is having results into a json file which will be read thanks to file_get_contents (be careful with the return type).
It will avoid the json_encode too.